FlatFilers/demo-agent
# feat: expand documents demo

Adds the following features to the documents demo space:

* Actions
* Ephemeral documents
* Embedded Sheets

<img width="1644" alt="Screenshot 2023-10-19 at 3 32 35 PM" src="https://github.com/FlatFilers/demo-agent/assets/3711771/7771851f-d240-4cce-8cb3-5b43dcc6b704">
<img width="1644" alt="Screenshot 2023-10-19 at 3 32 37 PM" src="https://github.com/FlatFilers/demo-agent/assets/3711771/e656a6b9-effb-4575-9844-f3eece7624b0">
<img width="1644" alt="Screenshot 2023-10-19 at 3 32 40 PM" src="https://github.com/FlatFilers/demo-agent/assets/3711771/a4e0e4d8-4520-4562-acfd-14d9f8159559">
<img width="1644" alt="Screenshot 2023-10-19 at 3 32 45 PM" src="https://github.com/FlatFilers/demo-agent/assets/3711771/d58e86e5-c5f7-4f6a-8c26-fe2f5c7711b5">


To test:
1. Pull down this branch
2. run `npx flatfile@latest develop src/demos/documents.ts`
3. Create a space
4. Check out all of the new documents that were added
